src/app/system/audit-trails/view-audit/view-audit.component.scss

```diff
@@ -0,0 +1,18 @@
+.container {
+    max-width: 37rem;
```

Comment:
Similar issue with indentation as specified above. Check in other files as well.